Resources/skeleton/crud/views/others/record_actions.html.twig.twig

```diff
@@ -1,17 +1,24 @@
-<ul class="record_actions">
-    <li>
-        <a href="{{ "{{ path('" ~ route_name_prefix ~ "') }}" }}">
-            Back to the list
-        </a>
-    </li>
-{% if ('edit' in actions) and (not hide_edit) %}
-    <li>
-        <a href="{{ "{{ path('" ~ route_name_prefix ~ "_edit', { 'id': " ~ entity_singularized ~ "."~ identifier ~" }) }}" }}">
-            Edit
-        </a>
-    </li>
-{% endif %}
-{% if ('delete' in actions) and (not hide_delete) %}
-    <li>{{ '{{ form(delete_form) }}' }}</li>
-{% endif %}
-</ul>
+    <ul class="record_actions">
+        <li>
+            <a href="{{ "{{ path('" ~ route_name_prefix ~ "_index') }}" }}">Back to the list</a>
+        </li>
+
+        {%- if ('edit' in actions) and (not hide_edit) %}
+
```

Comment:
In fact, I based on some templates that use blank line as [index.html.twig.twig](https://github.com/sensiolabs/SensioGeneratorBundle/blob/v2.5.3/Resources/skeleton/crud/views/index.html.twig.twig) or [others/actions.html.twig.twig](https://github.com/sensiolabs/SensioGeneratorBundle/blob/v2.5.3/Resources/skeleton/crud/views/others/actions.html.twig.twig) and it work. How would you have done ?